lib/pharos/phases/configure_master.rb

```diff
@@ -90,6 +91,7 @@ def push_kube_certs(certs)
           transport.file(path).write(contents)
           transport.exec!("sudo chmod 0400 #{path}")
         end
+        host.checks['ca_exists'] = true
```

Comment:
Are we using this?

Reply:
```
lib/pharos/phases/validate_host.rb
40:        raise Pharos::InvalidHostError, "Cannot change worker host role to master" if @host.master? && !@host.checks['ca_exists']
41:        raise Pharos::InvalidHostError, "Cannot change master host role to worker" if @host.worker? && @host.checks['ca_exists']
```

Reply:
I meant are we using it in this PR? How is this relevant?